.gitignore

```diff
@@ -42,6 +42,7 @@ install_manifest.txt
 *.xcworkspace
 
 # additional *nix generated files
+/ch
```

Comment:
Why `/ch` instead of just `ch` doesn't the leading slash only exclude the file if it is in the same directory as the `.gitignore`?